### DISCUSSION TITLE
How about build and release tauri-cli for more platform.

### DISCUSSION THREAD

**Discussion by liudonghua123** [tauri-apps/tauri#11380] [Ideas]
I am write a small [tauri-build](https://github.com/liudonghua123/tauri-build) project which build a multi-platform docker images for easy build tauri app.

I only find a x86_64 prebuild in https://github.com/tauri-apps/tauri/releases/tag/tauri-cli-v2.0.3 for linux. How about add more targets in the [ci](https://github.com/tauri-apps/tauri/blob/dev/.github/workflows/publish-cli-rs.yml) like i386, arm64 and armv7.

I have to build tauri-cli from source in my [Dockerfile](https://github.com/liudonghua123/tauri-build/blob/main/Dockerfile) currently. `cargo install tauri-cli` will spend a few hours in github action runner. Maybe this step could replace by downloading the prebuild tauri-cli binary for installing.

**Comment by FabianLars** [comment]
I agree that this would nice (not nice for our release ci times though), until then however, would using the npm cli be a valid alternative? Doing `npm i -g @tauri-apps/cli` would make it available via `tauri build` instead of `cargo tauri build` and the npm cli is built for more architectures.
